views/scss/item-comments.scss

```
@@ -0,0 +1,270 @@
+.item-comments-reminder {
+    font-size: 11px;
+    color: #666;
+    margin: 0 0 8px;
+}
+
+.item-comments-label {
+    position: absolute;
+    width: 1px;
+    height: 1px;
+    overflow: hidden;
+    clip: rect(0 0 0 0);
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)